### ISSUE TITLE
HTML entity &zeroWidthSpace; is rendered as text - [it is &ZeroWidthSpace;, still not rendered]

### ISSUE THREAD

**Issue by mcesar-rlacruz** [typora/typora-issues#4136]
Hi,

When the HTML entity `&zeroWidthSpace;` is entered, it is rendered as that exact text, not as a zero width space.

Entering it as `&#8203;` does work, though. Entities `&zwj;` and `&zwnj;` also work as expected.

v0.9.98(beta) on Windows10.

Thanks & regards.

**Comment by abnerlee** [comment]
Because HTML entity &zeroWidthSpace; does not exist? See following, tested in github:

source

```
When the HTML entity &zeroWidthSpace; is entered, it is rendered as that exact text, not as a zero width space.

Entering it as &#8203; does work, though. Entities &zwj; and &zwnj; also work as expected.
```

output:

When the HTML entity &zeroWidthSpace; is entered, it is rendered as that exact text, not as a zero width space.

Entering it as &#8203; does work, though. Entities &zwj; and &zwnj; also work as expected.

**Comment by mcesar-rlacruz** [comment]
Hi again,

I'm sorry that I mispelled the HTML entity, it is `&ZeroWidthSpace;` (with a capital Z). Here I wrote it inside double quotes "&ZeroWidthSpace;" (it will be not visible.

Regards.

**Comment by abnerlee** [comment]
a&ZeroWidthSpace;b

**Comment by kent93525** [comment]
​

**Comment by abnerlee** [comment]
fixed in v1.5.x

**Comment by kf45mc** [comment]
&ZeroWidthSpace;
